tests/testthat/examples/Test_2023_1/metadata.yml

```diff
@@ -64,13 +64,10 @@ dataset:
 locations:
   Atherton:
     description: Tropical rain forest vegetation.
-    elevation (m): 800
```

Comment:
Did you mean to modify this file?

Reply:
@ehwenk tests now passing. Just need to answer this and you can merge

Reply:
I'll go ahead and move the one line - when I changed it back I didn't realise that I'd resorted the rows.